ui/src/types/layout.ts

```diff
@@ -0,0 +1,5 @@
+import {TimeSeriesServerResponse} from 'src/types/series'
+
+export type GrabDataForDownload = (
```

Comment:
could this be named something to indicate that it is a function type?

Reply:
Would adding a `Handler` suffix be helpful?